odra-casper/rpc-client/src/casper_client/configuration.rs

```diff
@@ -66,18 +74,22 @@ impl CasperClientConfiguration {
         }
         (secret_keys, secret_key_paths)
     }
-}
 
-fn get_env_variable(name: &str) -> String {
-    std::env::var(name).unwrap_or_else(|err| {
-        crate::log::error(format!(
-            "{} must be set. Have you setup your .env file?",
-            name
-        ));
-        panic!("{}", err)
-    })
-}
+    fn get_env_variable(name: &str) -> String {
```

Comment:
I'd rather put it into `utils` module, or leave it as it was. No need to call with `Self::`

Reply:
What if I told you that I just deleted whole utils module?

Reply:
`odra-casper/rpc-client/src/utils.rs` is a thing, so maybe reuse the same method from there.